doc/metrics.md

```diff
@@ -151,12 +155,15 @@ scrape_configs:
       #      is not covered by the certificate (not in the SAN list)
       server_name: 'foo'
 ```
-
 ````{note}
-The `server_name` must be specified if the LXD server certificate does not contain the same host name as used in the `targets` list.
-To verify this, open `server.crt` and check the Subject Alternative Name (SAN) section.
+* The `scrape_interval` is assumed to be 15s by the Grafana Prometheus data source by default.
+If you decide to use a different `scrape_interval` value, you must change it in both the Prometheus configuration and the Grafana Prometheus data source configuration.
+Otherwise the Grafana `$__rate_interval` value will be calculated incorrectly and possibly cause a `no data` response in queries using it.
 
-For example, assume that `server.crt` has the following content:
+* The `server_name` must be specified if the LXD server certificate does not contain the same host name as used in the `targets` list.
+  To verify this, open `server.crt` and check the Subject Alternative Name (SAN) section.
+
+  For example, assume that `server.crt` has the following content:
 
```

Comment:
Please also indent the following lines (since they are part of the second bullet point). "Since the ..." should still be indented, the back ticks should not.